rmgpy/thermo/thermoengine.py

```diff
@@ -156,8 +156,17 @@ def evaluator(spc, solvent_name=''):
     """
     logging.debug("Evaluating spc %s ", spc)
 
-    spc.generate_resonance_structures()
-    thermo = generate_thermo_data(spc, solvent_name=solvent_name)
+    from rmgpy.molecule import Molecule
```

Comment:
Since this file is not cythonized, double check if this import can be moved

Reply:
Done!